store/store_download_test.go

```diff
@@ -80,6 +83,21 @@ func (s *storeDownloadSuite) SetUpTest(c *C) {
 			Factor:  1,
 		},
 	)))
+
+	buf, restore := logger.MockLogger()
+	s.AddCleanup(restore)
+	s.logbuf = buf
+
+	os.Setenv("SNAPD_DEBUG", "1")
+	s.AddCleanup(func() { os.Unsetenv("SNAPD_DEBUG")})
```

Comment:
The tests don't need this bit, I guess it was for debugging, so maybe we can remove it